ftw/news/browser/news_listing.py

```diff
@@ -174,17 +178,20 @@ def get_manager_and_assignments(self, manager_name):
 
         return managers_and_assignments
 
-    def get_items(self):
-        portlet = self.get_portlet()
-        if portlet:
-            return portlet.get_news(all_news=True)
+    def get_query(self):
+        if self.portlet:
+            portlet_query = self.portlet.get_query(all_news=True)
+            view_query = super(NewsListingPortlet, self).get_query()
+
+            if 'start' in view_query:
+                portlet_query['start'] = view_query['start']
+            return portlet_query
 
-        return []
+        return {}
```

Comment:
I decided to use the default query... of the inherited class --> `return super(NewsListingPortlet, self).get_query()
`